### PR TITLE
[RM Ch04] acceleration requirements 

### PR DESCRIPTION
**Pull request by walterkozlowski**

### CODE REVIEW

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -97,10 +96,10 @@ This section describes a set of explicit Cloud Infrastructure capabilities and p
...
-| e.cap.014 | GPU/NPU                                   | Yes/No | Hardware coprocessor. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type. |
+| e.cap.013 | SR-IOV over PCI-PT                        | Yes/No | Traditional SR-IOV. These Capabilities generally require hardware-dependent drivers be injected into workloads. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the relevant RA specification for the usage policy relevant to any needed hardware capability of this type.  |
+| e.cap.014 | GPU/NPU                                   | Yes/No | Hardware coprocessor. These Capabilities generally require hardware-dependent drivers be injected into workloads. As such, use of these features shall be governed by the applicable CNTT policy.lease consult the relevant RA specification for the usage policy relevant to any needed hardware capability of this type.  |
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter04.md:100]:
```suggestion
| e.cap.014 | GPU/NPU                                   | Yes/No | Hardware coprocessor. These Capabilities generally require hardware-dependent drivers be injected into workloads. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the relevant RA specification for the usage policy relevant to any needed hardware capability of this type.  |
```
Typo
(or without the "please")

File: doc/ref_model/chapters/chapter04.md [hunk 2]
```diff
@@ -97,10 +96,10 @@ This section describes a set of explicit Cloud Infrastructure capabilities and p
...
 | e.cap.015 | SmartNIC                                  | Yes/No | Network Acceleration. SmartNICs that do not utilise PCI-PT are not subject to the CNTT principles, nor any related policies or prohibitions. |
-| e.cap.016 | FPGA/other Acceleration H/W               | Yes/No | Non-specific hardware. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type. |
+| e.cap.016 | FPGA/other Acceleration H/W               | Yes/No | Non-specific hardware. These Capabilities generally require hardware-dependent drivers be injected into workloads. As such, use of these features shall be governed by the applicable CNTT policy. lease consult the relevant RA specification for the usage policy relevant to any needed hardware capability of this type.  |
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter04.md:102]:
```suggestion
| e.cap.016 | FPGA/other Acceleration H/W               | Yes/No | Non-specific hardware. These Capabilities generally require hardware-dependent drivers be injected into workloads. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the relevant RA specification for the usage policy relevant to any needed hardware capability of this type.  |
```
Typo
(or without the "please")

**Review by ulikleber — APPROVED** [review]
when the typos are fixed.

**Review by tomkivlin — APPROVED** [review]

**Comment by tomvanpelt** [comment]
I wonder whether this should include the similar note in table 4-17 and section 4.2.7.3 on forward compatibility that both refer to the exception list in the annex as well or would those be updated through a separate pull request?

**Comment by walterkozlowski** [comment]
typos fixed as per comments by @ulikleber .

**Review by tomvanpelt — APPROVED** [review]

**Pull request merged**